src/python/pants/backend/codegen/protobuf/scala/ScalaPBShim.scala

```diff
@@ -0,0 +1,123 @@
+/*
+ * Copyright 2021 Pants project contributors (see CONTRIBUTORS.md).
+ * Licensed under the Apache License, Version 2.0 (see LICENSE).
+ */
+
+package org.pantsbuild.backend.scala.scalapb
+
+import java.util.jar.JarInputStream
+import java.io.{File, FileInputStream}
+import java.net.URLClassLoader
+import protocbridge.{ProtocBridge, ProtocCodeGenerator, ProtocRunner, SandboxedJvmGenerator}
+import scalapb.ScalaPbCodeGenerator
+
+
+// Derived in part from ScalaPBC under Apache License v2.0. Introduced errors are mine.
```

Comment:
Would be good to include the SHA you forked from.

Reply:
Done.